samples/unit-tests/rangeselector/buttons-range-state/demo.js

```diff
@@ -1,3 +1,83 @@
+QUnit.test('Range selector buttons general tests.', function (assert) {
```

Comment:
Actually we don't need a unit test for the button row, because the visual test (stock/rangeselector/styling) picks it up. Instead, we could use a unit test for the actual problem, which lies in the button function mutating its options. So instead of loading a full stock chart, create a much faster and to-the-point test on the `SVGRenderer.button` itself: https://jsfiddle.net/highcharts/hnt7g9zL/